scripts/install_linux.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
+# This script detects the current operating system architecture and installs the appropriate version of Ollama
+
+set -eu
+
+# Check for jq, systemd, and systemctl dependencies
+for cmd in "jq" "systemd" "systemctl"; do
+    if ! command -v $cmd > /dev/null 2>&1; then
+        echo "Error: $cmd is not installed, and this script requires it. Please install $cmd and try again."
```

Comment:
I really like these utility functions for logging and errors which might be useful here

```bash
status() { echo ">>> $*" >&2; }
error() { status "ERROR: $*"; exit 1; }
```